drivers/usb/device/usb_dc_stm32.c

```diff
@@ -264,6 +264,16 @@ static int usb_dc_stm32_clock_enable(void)
 	return 0;
 }
 
+static void usb_dc_stm32_clock_disable(void)
+{
+	struct device *clk = device_get_binding(STM32_CLOCK_CONTROL_NAME);
+	struct stm32_pclken pclken = {
+		.bus = DT_USB_CLOCK_BUS,
+		.enr = DT_USB_CLOCK_BITS,
+	};
+	clock_control_off(clk, (clock_control_subsys_t *)&pclken);
+}
+
```

Comment:
Check PR #16026. It creates a `static struct stm32_pclken pclken` and uses it for both  `clock_control_on` and `clock_control_off`.